src/hooks/useInitializeWallet.ts

```diff
@@ -127,27 +127,30 @@ export default function useInitializeWallet() {
 
         if (!(isNew || isImporting)) {
           await loadGlobalEarlyData();
-          logger.sentry('loaded global data...');
+          logger.log('loaded global data...');
         }
 
         await dispatch(settingsUpdateAccountAddress(walletAddress));
-        logger.sentry('updated settings address', walletAddress);
+        logger.log('updated settings address', { walletAddress });
 
         // Newly created / imported accounts have no data in localstorage
         if (!(isNew || isImporting)) {
           await loadAccountData(network);
-          logger.sentry('loaded account data', network);
+          logger.log('loaded account data', { network });
         }
-        if (!switching) {
+
+        try {
           hideSplashScreen();
+          logger.log('Hide splash screen');
```

Comment:
we have to places where we call the `hideSplashScreen` - here and on line 176. the `isSwitching` was to make the experience faster for folks who don't need to initialize the remaining account data - perhaps we should just remove the `hideSplashScreen here on line 143 and keep the later one